gui/wxpython/startup/guiutils.py

```diff
@@ -279,44 +280,66 @@ def rename_location_interactively(guiparent, grassdb, location):
     return newlocation
 
 
-def delete_mapset_interactively(guiparent, grassdb, location, mapset):
+def delete_mapset_interactively(guiparent, db_loc_mapset):
```

Comment:
This should return True/False, see #789.

Reply:
I rebased and solved conflict with master because of #789 and considered this as well.

Reply:
The merge ended up ignoring the return value, fixed.